client/src/partials/login/login.js

```diff
@@ -77,12 +74,14 @@ function LoginController(AppCache, Session, Languages, Projects) {
     })
     .catch(function (response) {
 
-      // bind the $http error to the view
-      vm.error = response.data;
+      // if the user has tried too many times, display a fatal error working for
+      // ten seconds.
+      if (maxAttempts <= attempts++) {
+        return Notify.fatal('AUTH.TOO_MANY_TRYS', 10000);
```

Comment:
I would reserve the use of the `fatal` style for errors that are beyond the users control, this would mean that whenever you see dark red it probably means a system administrator has to be involved. Whilst this use case is could be argued to be that way the user can still just put in their correct password. 

What do you think?

Too many attempts could potentially ramp up from `warn` to `danger` although I wouldn't have a problem with it being `danger` uniformly.